### PR TITLE
D3 596 597 598 | Interface changes

### PR DESCRIPTION
**Pull request by Asmadek**
# Description

https://soramitsu.atlassian.net/browse/D3-596
https://soramitsu.atlassian.net/browse/D3-597
https://soramitsu.atlassian.net/browse/D3-598

#### Works done
- [x] Add caption to portfolio history chart 
- [x] Add loading to portfolio history chart and assets history chart
- [x] Change position of tooltips in wallet and dashboard headers

# How to check
1. `yarn` to install all dependencies
2. `yarn serve` to run application

### CODE REVIEW

File: src/store/Dashboard.js [hunk 1]
```diff
@@ -176,23 +181,36 @@ const mutations = {
...
+  },
 
   [types.GET_PRICE_BY_FILTER_SUCCESS] (state, data) {
     Vue.set(state.assetChart, 'data', data)
+    Vue.set(state.assetChart, 'isLoading', false)
   },
 
   [types.GET_PRICE_BY_FILTER_FAILURE] (state, err) {
+    if (state.assetChart) {
+      state.assetChart.isLoading = false
```
Comment (Pobepto) [inline src/store/Dashboard.js]: Replace with ```Vue.set()```. I guess this `if` statement is unnecessary.

File: src/store/Dashboard.js [hunk 2]
```diff
@@ -176,23 +181,36 @@ const mutations = {
...
+    Vue.set(state.portfolio, 'isLoading', true)
+  },
 
   [types.GET_PORTFOLIO_HISTORY_SUCCESS] (state, data) {
     Vue.set(state.portfolio, 'assetsHistory', data)
+    Vue.set(state.portfolio, 'isLoading', false)
   },
 
   [types.GET_PORTFOLIO_HISTORY_FAILURE] (state, err) {
+    if (state.portfolio) {
```
Comment (Pobepto) [inline src/store/Dashboard.js]: Unnecessary `if`. Also replace with ```Vue.set()```

File: src/components/Settings/SettingsPage.vue [hunk 3]
```diff
@@ -92,6 +92,7 @@
                   <div class="settings-item_row" :key="index">
                     <span @click="() => doCopy(pubKey)" class="settings-item_row-key pointed">{{ pubKey | substrKey}}</span>
                     <el-button
+                      v-if="accountSignatories.length != accountQuorum"
```
Comment (Pobepto) [inline src/components/Settings/SettingsPage.vue]: Error after rebase. This line should not exist, need to rebase again.

File: src/store/Dashboard.js (hunk 1 above)
Comment (Asmadek) [reply src/store/Dashboard.js]: testErrorHandling doesn't work without if. Or maybe we need to change testErrorHandling function?

Comment (Pobepto) [reply src/store/Dashboard.js]: Hm, rewrite this function seems to be good idea. Maybe provide `state` as second argument? Also don't access to types directly, please use object `types` that contains them.

`'GET_PORTFOLIO_HISTORY_FAILURE'` -> `types.GET_PORTFOLIO_HISTORY_FAILURE`

**Pull request merged**